proxy/server.py

```diff
@@ -279,15 +285,22 @@ def do_POST(self):
         global proxystats
         contenttype = 'application/json'
         message = '{"error": "Invalid Request"}'
-        if self.path.startswith('/control'):
+
+        # Get the request path and subtract the base URL. This is intentionally doing a replace on the URL we check
+        # instead of using the prefix inside of the checks to maintain the ability to use the regular URLs in addition to
+        # the proxied URLs, so Telegraf configs or existing integrations don't need to be modified.
+        request_path = self.path
+        request_path = request_path.replace(api_base_url, "/")
```

Comment:
Something about this bothers me. The real intent is to replace just the prefix URI (e.g. `/pypowerwall/`) but the replace function will replace any occurence api_base_url within the string of self.path. Your example works because there aren't any URI's that use that, but if someone set this to `/system_status/` for example, it is going to wreck havoc with several APIs. I wonder if we should do something like this instead:
```python
        request_path = self.path
        if request_path.startswith(api_base_url):
            request_path = "/" + request_path[len(api_base_url):]
```

Reply:
Good thought, made that change in both places